src/Compilers/CSharp/Portable/Symbols/PublicModel/RangeVariableSymbol.cs

```diff
@@ -25,7 +23,8 @@ protected override void Accept(SymbolVisitor visitor)
             visitor.VisitRangeVariable(this);
         }
 
-        protected override TResult Accept<TResult>(SymbolVisitor<TResult> visitor)
+        protected override TResult? Accept<TResult>(SymbolVisitor<TResult> visitor)
```

Comment:
Should the override below (at line 32) have an annotation on return type as well?

Reply:
No, that visitor implementation was added later and correctly produces a default value that matches the declared type.